lib/eunit/include/eunit.hrl

```diff
@@ -166,6 +166,28 @@
 %% This is mostly a convenience which gives more detailed reports.
 %% Note: Guard is a guarded pattern, and can not be used for value.
 -ifdef(NOASSERT).
+-define(assertReceive(Guard, Timeout), ok).
+-else.
+-define(assertReceive(Guard, Timeout),
+	begin
+	((fun () ->
+	    receive (Guard) ->
+            ok
+        after Timeout ->
+		    erlang:error({assertReceive_timeout,
```

Comment:
Please move this to line 176, after the "after Timeout ->" and the ok on line 175 to 174. As you see in the rest of the file there shouldn't be a new line in these cases.